### ISSUE TITLE
Weird top layer bellow numbers

### ISSUE THREAD

**Issue by 0x3333** [supermerill/SuperSlicer#2882]
### What happened?

I'm going to print a screw jig(to sort them) and the top layer has some artifacts below the numbers:

![image](https://user-images.githubusercontent.com/289289/173664708-f9cfddc8-527c-41a7-9756-383923e459b9.png)

Here is the sliced version:

![image](https://user-images.githubusercontent.com/289289/173664755-81174357-cdfd-4d28-b1ad-c566f48dabce.png)

Here is a zoom:
![image](https://user-images.githubusercontent.com/289289/173664860-3ad4ddaf-2639-479c-920d-acbd94a6fc47.png)

This same STL works OK in PrusaSlicer:
![image](https://user-images.githubusercontent.com/289289/173666355-acac688e-3b41-41d0-9e06-c414f392b80b.png)

Is there any possibility that this is an issue with my config?

Ps.: I'm using git master version, compiled in my MacBook Pro M1.

### Project file & How to reproduce

[screwmeasuringtool_v1_with_m2.5.3mf.zip](https://github.com/supermerill/SuperSlicer/files/8902969/screwmeasuringtool_v1_with_m2.5.3mf.zip)

### Version

2.3.57.12

### Operating system

MacOS Monterey

### Printer model

Voron 2.4 with StealthBurner

**Comment by neophyl** [comment]
Don’t even need to open your project. Turn off single perimeter on top layers option. That’s not an option PS has which is why it slices differently.

**Comment by 0x3333** [comment]
Damn, I tried changing extrusion width and several settings, but this one I didn't, as this isn't, technically, the top layer.

Thanks for the eagle eyes.

**Comment by neophyl** [comment]
You can also change the Minimum width value next to the setting.  Dropping it to 150% gets rid of the slicing artefacts below the small top raised numbers but still leaves some in the bottom of the recessed numbers.  Dropping it further removes more.  100% looks ok to me with some very minor ones located where they wont be noticeable.

**Comment by 0x3333** [comment]
You made it clear in your last comment, but I'd like to register here that I read the info about `min_width_top_surface` and now it makes total sense. It is actually not "artifacts" it is trying to do what it should do. This thing is impressive. Thanks again.

**Comment by neophyl** [comment]
Yeah it is :)  the single perimeter option is 'relatively' newish and we have had a fair few threads where users have reported issues that when looked at trace back to the function.  It's one reason I picked up on it from your screen caps.  And yes its just doing what its supposed to but many times it looks like a bug, especially to those that aren't as familiar with the software and where they do a comparison with PS.  

Basically there are a few settings that are goto's to check first if there are reported slicing problems.  Things like thin wall for example, which has always been problematic.
